manifest.json

```diff
@@ -20,5 +20,14 @@
   },
   "background": {
     "service_worker": "src/contextMenu.js"
+  },
+  "commands": {
+    "change-host-name": {
+      "suggested_key": {
+        "default": "Alt+Q",
+        "mac": "Alt+Q"
```

Comment:
Since there is no Alt key in mac, is it possible to add `MacCtrl ` instead?

Reply:
Yeah sure. It's also worth mentioning that by navigating to chrome://extensions/shortcuts the user can change it to whatever

Reply:
i will add it in the readme, i was thinking also we could add a shortcut to open in localhost in new tab since for me that's the most used case. I can add it in later time